source/features/conversation-activity-filter.tsx

```diff
@@ -99,8 +110,10 @@ async function handleSelection({target}: Event): Promise<void> {
 	);
 
 	// Update the state of the other dropdown
-	select(`.${dropdownClass} [aria-checked="false"][data-value="${currentSetting}"]`)!.setAttribute('aria-checked', 'true');
-	select(`.${dropdownClass} [aria-checked="true"]:not([data-value="${currentSetting}"])`)!.setAttribute('aria-checked', 'false');
+	if (pageDetect.isConversation()) {
```

Comment:
flipped the condition you suggested is because `if not condition` is less readable than now.